doc/getting_started/index.rst

```diff
@@ -471,71 +471,142 @@ Install a Toolchain
 A toolchain provides a compiler, assembler, linker, and other programs required
 to build Zephyr applications.
 
+The Zephyr Software Development Kit (SDK) contains toolchains for each of
+Zephyr's supported architectures. It also includes additional host tools, such
+as custom QEMU and OpenOCD.
+
 .. tabs::
 
    .. group-tab:: Ubuntu
 
-      The Zephyr Software Development Kit (SDK) contains toolchains for each of
-      Zephyr's supported architectures. It also includes additional host tools,
-      such as custom QEMU binaries and a host compiler.
-
-      |p|
-
-      #. Download the `latest SDK installer
+      #. Download and verify the `latest Zephyr SDK bundle
          <https://github.com/zephyrproject-rtos/sdk-ng/releases>`_:
 
          .. code-block:: bash
 
             cd ~
-            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.13.2/zephyr-sdk-0.13.2-linux-x86_64-setup.run
+            wget https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/zephyr-sdk-0.14.0_linux-x86_64.tar.gz
+            wget -O - https://github.com/zephyrproject-rtos/sdk-ng/releases/download/v0.14.0/sha256.sum | shasum --check --ignore-missing
 
-      #. Run the installer, installing the SDK in :file:`~/zephyr-sdk-0.13.2`:
+      #. Extract the Zephyr SDK bundle archive:
 
          .. code-block:: bash
 
-            chmod +x zephyr-sdk-0.13.2-linux-x86_64-setup.run
-            ./zephyr-sdk-0.13.2-linux-x86_64-setup.run -- -d ~/zephyr-sdk-0.13.2
+            tar xvf zephyr-sdk-0.14.0_linux-x86_64.tar.gz
```

Comment:
Shouldn't users go to the folder first, before executing this command?

Reply:
Applies to macOS too

Reply:
That was written with the premise that the user is installing the Zephyr SDK under `~`.

Note that the previous step "Download and verify the latest Zephyr SDK bundle" already specifies `cd ~`.

Reply:
mainly because using `~/.local` for installing programs is arguably controversial and not really a "standard practice."

e.g. https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html